docs/skopeo-copy.1.md

```diff
@@ -149,6 +149,18 @@ Let's say out of 3 layers that the image `docker.io/library/nginx:1.17.8` is mad
 skopeo  copy --encryption-key jwe:./public.key --encrypt-layer 1 oci:local_nginx:1.17.8 oci:try-encrypt:encrypted
 ```
 
+## NOTES
+
+In the context of `signatures`, this refers to `container signatures` as implemented by the [github.com/containers/image/signature](https://github.com/containers/image/tree/main/signature) package.
+
+```text
+The signature provides an end-to-end authenticated claim that a container image has been approved by a specific party (e.g. the creator of the image as their work, an automated build system as a result of an automated build, a company IT department approving the image for production) under a specified identity (e.g. an OS base image / specific application, with a specific version).
```

Comment:
nits for consideration
```suggestion
The signature provides an end-to-end authenticated claim that a specific party has approved a container image (e.g., the creator of the image as their work, an automated build system as a result of an automated build, a company IT department approving the image for production) under a specified identity (e.g., an OS base image / specific application, with a specific version).
```

Reply:
Thanks, as per above comment, I'm slimming down the changes so the verbiage has gone gone gone 🤣